phoenicis-containers/pom.xml

```diff
@@ -64,5 +64,10 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.phoenicis</groupId>
+            <artifactId>phoenicis-library</artifactId>
+            <version>5.0-SNAPSHOT</version>
```

Comment:
I guess you do not want to duplicate the GUI for this? So it probably would be enough to jump to the Containers tab.